test_pull_requests

```diff
@@ -847,12 +759,11 @@ end
       sleep 10
       params = { :commit_message => "Merged by #{@properties['bot_github_user']}" }
       res = nil
-      num_tries = 5
+      num_tries = 3
```

Comment:
We're getting retry logic for free from the middleware stack, so we don't need to rerun this bit as often as before.